adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -640,6 +640,17 @@ public Frequency getFrequency(final String tripId, final Integer startTime) {
         return frequencyPerTripIdStartTime.get(Frequency.getFrequencyMappingKey(tripId, startTime));
     }
 
+    /**
+     * Return an unmodifiable map of Frequency objects representing all the rows from frequencies.txt. Entities are
+     * mapped on values of fields trip_id and start_time of file `frequencies.txt`.
```

Comment:
So this is a composite key? I'd add to the Javadocs explicitly how to create the key (link to the utility method, if one exists) so future developers know.